recipe/meta.yaml

```diff
@@ -37,17 +37,16 @@ requirements:
     - {{ posix }}make
     - {{ posix }}coreutils         # [win]
     - {{ posix }}zip               # [win]
-    - llvm-openmp   # [osx]
   host:
     - r-base
     - r-rcpp >=0.11.0
     - libblas
     - liblapack
+    - llvm-openmp                  # [osx]
```

Comment:
Move this to `build`

Reply:
Still results in [`llvm-openmp=15` in the test environment](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=641406&view=logs&j=9c5ef928-2cd6-52e5-dbe6-9d173a7d951b&t=20c71c51-4b27-578b-485d-06ade2de1d00&l=5659).

Reply:
It's fine to have `llvm-openmp=15` in the test env. (`llvm-openmp=15` satisfies `llvm-openmp>=14`)